aplib.net-demo/Assets/Scripts/Tiles/TSection.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+
+namespace Assets.Scripts.Tiles
+{
+    /// <summary>
+    /// Represents a T-section tile.
```

Comment:
It might be a little overkill, but it would be nice to have a simple visual cue as to what the tile looks like,
like:
```suggestion
    /// Represents a T-section tile.
    ///  __ __ __
    /// |________|
    ///  __    __
    /// |__|  |__|
```
same for the other tiles